Corzbank.Web/ClientApp/src/styles.css

```diff
@@ -14,3 +14,13 @@ code {
   background-color: #1b6ec2;
   border-color: #1861ac;
 }
+body{
+  display: block;
+  text-align: -webkit-center;
+  background-color: #24203A;
+}
+.main{
```

Comment:
what is main selector? Is it used somewhere?

Reply:
resolved